src/App.tsx

```diff
@@ -1,10 +1,72 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
+
+import { TodoInfo } from './components/todoinfo/todoinfo';
+import { Header } from './components/header/header';
+import { Footer } from './components/footer/footer';
+import { Error } from './components/error/error';
+
 import { UserWarning } from './UserWarning';
+import { client } from './utils/fetchClient';
+
+import { Todo } from './types/Todo';
+import { Filter } from './components/filter/filter';
 
-const USER_ID = 0;
+const USER_ID = 10683;
+const URL = `/todos?userId=${USER_ID}`;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<null | Todo[]>(null);
+  const [filterValue, setFilterValue] = useState(Filter.All);
+  const [isAddError, setIsAddError] = useState(false);
+  const [isDeleteError, setIsDeleteError] = useState(false);
+  const [isUpdateError, setIsUpdateError] = useState(false);
```

Comment:
It would be better to create an enum for `ErrorType` and set the error type instead of using different flags